### PR TITLE
CIRC-4456 Clarify that zpool must exist

### PR DESCRIPTION
**Pull request by esproul**

### CODE REVIEW

File: content/circonus/on-premises/installation/installation.md [hunk 1]
```diff
@@ -35,6 +35,11 @@ appendix giving a brief tutorial on ZFS setup. Note, however, that the final
 step of the appendix, which refers to IRONdb setup, is not required for
 Circonus Inside. **Do not install any IRONdb packages.**
 
+The `data_storage` role expects the ZFS storage pool (zpool) to exist prior
+to starting the installation process. The zpool may be named as desired, and
+all hosts in the role must have the [zfs_dataset_base](#machinfo-attributes)
```
Comment (bdunavant) [inline content/circonus/on-premises/installation/installation.md:40]: Unclear.  I have to have zfs_dataset_base set, but with what value?  The name of the zpool?

Comment (esproul) [reply content/circonus/on-premises/installation/installation.md:40]: It's explained in the description of the attribute that this links to.

Comment (esproul) [reply content/circonus/on-premises/installation/installation.md:40]: Attempted to clarify.

**Review by bdunavant — APPROVED** [review]
LGTM

**Pull request merged**